ethereum/core/src/test/java/org/hyperledger/besu/ethereum/trie/diffbased/bonsai/BonsaiSnapshotIsolationTests.java

```diff
@@ -152,6 +152,8 @@ public void assertCloseDisposesOfStateWithoutCommitting() {
       assertThat(res.isSuccessful()).isTrue();
       shouldCloseSnapshot.persist(oneTx.getHeader());
 
+      // TODO: full flat strategy will not be updated with these values, only the trie
+      //       determine how we want to make this assertion for full flat
```

Comment:
I think we can use an updater in order to fix the test

Reply:
I thought about it and realized it will degrade the performance of the 'normal' use case for frozen/cached worldstates.  I think this note here should be sufficient to explain why we are using PARTIAL for the test - removed from TODO.